src/tests/ftest/launch.py

```diff
@@ -956,13 +957,13 @@ def archive_files(destination, host_list, source_files):
     #
     # pylint: disable=bad-continuation
     commands = [
-        "set -eu",
+        "set -ux",
         "rc=0",
         "copied=()",
-        "for file in $(ls {})".format(source_files),
+        "for file in $(ls -d {})".format(source_files),
         "do ls -sh $file",
-        "/lib/daos/TESTING/ftest/cart/util/cart_logtest.py $file",
-        "if scp $file {}:{}/${{file##*/}}-$(hostname -s)".format(
+        "/lib/daos/TESTING/ftest/cart/cart_logtest.py $file",
+        "if scp -r $file {}:{}/${{file##*/}}-$(hostname -s)".format(
```

Comment:
As these `$file` should only be a `*.log*` file -and not a directory - is this really needed?

Reply:
There are tests launched via [orterun](https://www.open-mpi.org/doc/v4.0/man1/orterun.1.php) in the CaRT tests which log to test directories.  Example Jenkins run:

https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-3139/169/artifact/Functional/job-2020-09-29T04.34-ce885d7-cart-cart_corpc_one_node/daos_logs/corpc_errors_no_sep_output.orterun_log-wolf-102vm2/1/

So the log file format looks something like this:

```
./archive/Functional/job-@datestamp@-cart-@test_name@/daos_logs/@test_name@_@env_name@.orterun_log-@node_name@/1/rank.@rank_no@/stdout
./archive/Functional/job-@datestamp@-cart-@test_name@/daos_logs/@test_name@_@env_name@.orterun_log-@node_name@/1/rank.@rank_no@/stderr
```

Concrete example:

```
./archive/Functional/job-2020-09-29T05.13-ca7c348-cart-cart_iv_one_node/daos_logs/iv_basic_sep_output.orterun_log-wolf-102vm2/1
./archive/Functional/job-2020-09-29T05.13-ca7c348-cart-cart_iv_one_node/daos_logs/iv_basic_sep_output.orterun_log-wolf-102vm2/1/rank.2
./archive/Functional/job-2020-09-29T05.13-ca7c348-cart-cart_iv_one_node/daos_logs/iv_basic_sep_output.orterun_log-wolf-102vm2/1/rank.2/stdout
./archive/Functional/job-2020-09-29T05.13-ca7c348-cart-cart_iv_one_node/daos_logs/iv_basic_sep_output.orterun_log-wolf-102vm2/1/rank.2/stderr
./archive/Functional/job-2020-09-29T05.13-ca7c348-cart-cart_iv_one_node/daos_logs/iv_basic_sep_output.orterun_log-wolf-102vm2/1/rank.1
./archive/Functional/job-2020-09-29T05.13-ca7c348-cart-cart_iv_one_node/daos_logs/iv_basic_sep_output.orterun_log-wolf-102vm2/1/rank.1/stdout
./archive/Functional/job-2020-09-29T05.13-ca7c348-cart-cart_iv_one_node/daos_logs/iv_basic_sep_output.orterun_log-wolf-102vm2/1/rank.1/stderr
./archive/Functional/job-2020-09-29T05.13-ca7c348-cart-cart_iv_one_node/daos_logs/iv_basic_sep_output.orterun_log-wolf-102vm2/1/rank.0
./archive/Functional/job-2020-09-29T05.13-ca7c348-cart-cart_iv_one_node/daos_logs/iv_basic_sep_output.orterun_log-wolf-102vm2/1/rank.0/stdout
./archive/Functional/job-2020-09-29T05.13-ca7c348-cart-cart_iv_one_node/daos_logs/iv_basic_sep_output.orterun_log-wolf-102vm2/1/rank.0/stderr
...
```